src/style.css

```
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.container {
+  display: flex;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  line-height: 300px;
+  /* width: 100%; */
+  text-align: center;
+}
+
+.block:nth-child(1) {
+  background: red;
+}
+
+.block:nth-child(2) {
+  background: rgb(204, 0, 0);
+}
+
+.block:nth-child(3) {
+  background: rgb(153, 0, 0);
+
+}
+
+.block:nth-child(4) {
+  background: rgb(102, 0, 0);
+
+}
+
+.block:nth-child(5) {
+  background: rgb(51, 0, 0);
+
+}
+
+.block:nth-child(6) {
+  background: black;
+}
+
+@media screen and (min-width: 750px) {
+  .block {
+    width: 33.3%;
+  }
+
+  .block:nth-child(4) {
+    order: 6;
+  }
+
+  .block:nth-child(5) {
+    order: 5;
+  }
+
+  .block:nth-child(6) {
+    order: 4;
+  }
+}
+
+@media screen and (max-width: 750px) {
+  .block {
+    width: 50%;
```

```suggestion
    flex-basis: 50%;
```
Так як ми працюємо з флексбоксом, варто використовувати його властивості